lib/pipeline.js

```diff
@@ -499,6 +487,50 @@ class PipelineModel extends BaseModel {
         return admin;
     }
 
+    /**
+     * This function deletes admins who does not have proper
+     * GitHub permission, and returns an proper admin.
+     * @method getFirstAdmin
+     * @return {Promise}
+     */
+    async getFirstAdmin() {
+        /* eslint-disable no-restricted-syntax */
+        for (const username of Object.keys(this.admins)) {
+            try {
+                // eslint-disable-next-line no-await-in-loop
+                return await new Promise((resolve, reject) => {
```

Comment:
why we want to use await here? don't think it's needed. also don't think `async` this function is needed, if we're not going to await it any where.

Reply:
Thank you. It seems to work properly, and the code is nice. I fixed.